core/lib/zksync_core/src/metrics.rs

```diff
@@ -183,3 +183,13 @@ pub(crate) struct ExternalNodeMetrics {
 
 #[vise::register]
 pub(crate) static EN_METRICS: vise::Global<ExternalNodeMetrics> = vise::Global::new();
+
+#[derive(Debug, Metrics)]
+#[metrics(prefix = "house_keeper")]
+pub(crate) struct HouseKeeperMetrics {
+    pub prover_job_archived_amount: Counter,
```

Comment:
nit: IMO the `_amount` in the end doesn't improve readability since it's already in metrics/counter context. Note that this name also ends up in prometheus/grafana, where it's concatenated with all the prefixes - already long.